seed/data_importer/match.py

```diff
@@ -364,6 +376,30 @@ def states_to_views(unmatched_state_ids, org, cycle, StateClass):
     return list(set(processed_views)), duplicate_count, new_count + matched_count
 
 
+def link_views(merged_views, ViewClass):
```

Comment:
This is the most notable change in this file. The other changes are mostly from a refactor of matching criteria getters and variable/function renaming.